events/blocks/event-agenda/event-agenda.js

```diff
@@ -72,18 +72,29 @@ export default async function init(el) {
 
     const venueImageCol = createTag('div', { class: 'venue-img-col' });
     el.classList.add('blade');
-    const h2 = el.querySelector('h2');
-    agendaItemsCol.prepend(h2);
 
     venueImageCol.append(createOptimizedPicture(imgUrl, venueImage.altText || '', false));
     container.append(venueImageCol);
   }
 
+  const h2 = el.querySelector('h2');
```

Comment:
This DOM structure change seems to cause some visual regressions. I do like how you are optimising to use more CSS to cover different cases. Probably just gotta make sure the padding on both sides are consistent.
<img width="1533" alt="Screenshot 2025-02-21 at 6 53 10 PM" src="https://github.com/user-attachments/assets/244198b1-7377-4e31-b576-d6123000b024" />
<img width="1496" alt="Screenshot 2025-02-21 at 6 53 20 PM" src="https://github.com/user-attachments/assets/b94e65b0-787a-4453-aaef-2c080ec61694" />